integrations/walkme/lib/index.js

```diff
@@ -67,7 +68,8 @@ WalkMe.prototype.initialize = function() {
     sriSuffix = 'private_';
   }
 
-  var url = 'https://cdn.walkme.com/users/' + walkMeSystemId + env + '/walkme_' + sriSuffix + walkMeSystemId + '_https.js';
+  var bucket = (this.options.customDirectory) ? this.options.customDirectory : 'users';
+  var url = `https://cdn.walkme.com/${bucket}/${walkMeSystemId}/${env}/walkme_${sriSuffix}${walkMeSystemId}_https.js`;

```

Comment:
@paco-walkme - I failed to catch this earlier but after reviewing this PR again my teammate we noticed that you are using template literals here. The integrations in this repo don't support the modern ES6 syntax. Could you change this back to string concatenation with `+`?

Reply:
@varadarajan-tw thanks for noticing! I've pushed a fix for it